packages/tooltip/src/Tooltip.tsx

```diff
@@ -259,38 +260,48 @@ function Tooltip({
 
   const createTriggerProps = useCallback(
     (triggerEvent: TriggerEvent, triggerProps?: any) => {
-      if (triggerEvent === TriggerEvent.Hover) {
-        return {
-          ref: setTriggerNode,
```

Comment:
This PR removes `ref` prop since it was causing some tests to fail. I may want to look into that deeper in the future

Reply:
Everything seems to be working, but lines 317-318 have a comment related to the triggerNode. Are we able to remove the usage of triggerNode if we aren't ever setting it?

Reply:
Removed references to `triggerNode` (see 312-320). Doesn't seem like it's needed elsewhere